syntax = "proto3";
package grpcnative;
option java_package = "sb.rpc.grpc4";
option java_outer_classname = "OrderServiceProto4";
option java_multiple_files = true;

// The greeting service definition.
service OrderService4 {
    // Sends a greeting
    rpc getOrderDetail (OrderRequest4) returns (OrderDetail4) {}
}

message OrderRequest4 {
    string orderId = 1;
}

message OrderDetail4 {
    string name = 1;
    uint64 date = 2;
    double price = 3;
    string buyer = 4;
    string seller = 5;
}